### ISSUE TITLE
Error installing on windows

### ISSUE THREAD

**Issue by nicstrong** [keybase/client#4158]
Running the lastest installer i get the following error:

![capture](https://cloud.githubusercontent.com/assets/54459/18192129/66e872e2-7125-11e6-98f4-297f5b417017.PNG)

Have tried removing the existing install. Still the same problem. Logs attached.

[Keybase_20160902155720_000_KeybasePrograms.log.txt](https://github.com/keybase/client/files/451135/Keybase_20160902155720_000_KeybasePrograms.log.txt)
[Keybase_20160902155720.log.txt](https://github.com/keybase/client/files/451136/Keybase_20160902155720.log.txt)

**Comment by maxtaco** [comment]
Cc: @zanderz

**Comment by zanderz** [comment]
This is an interesting case where stopping the running programs failed (`taskkill /F /IM kbfsdokan.exe`). What version is currently installed? It doesn't seem like removing made any difference. Maybe there is still some older version running under that? After uninstalling from the control panel, use task manager or the `taskkill` command to make sure `kbfsdokan.exe` and all instances of `keybase.exe` are stopped, then run the newer installer.

**Comment by nicstrong** [comment]
I cannot see the kbfsdokan.exe running in process explorer (taskkill returned not found).

The directory C:\Users\nics\AppData\Roaming\Keybase folder is empty. There was originally an folder called guicache or something similar which i removed. I can create an delete files without any errors in that folder.

**Comment by zanderz** [comment]
oh, maybe that wasn't it, then. The only other thing I can think of is that the `%appdata%\Keybase` directory was created with different permissions by a previous installation - what if you delete it first? Otherwise I'm not sure what to try, most installs write to that location no problem. Is this a corporate machine with some group policy in effect?

**Comment by nicstrong** [comment]
Did some digging further on this, it was strange as I could write files to the folder but not copy exes. I spoke to my admin and it turns out he applied blocking rules through our anti-virus since I last installed since some people got hit by cryptolocker. If you look at [this](https://kc.mcafee.com/resources/sites/MCAFEE/content/live/PRODUCT_DOCUMENTATION/25000/PD25203/en_US/Ransomware_Update_RevJ.pdf) link the recommend blocking appdata/roaming for *.exe.

Is there any reason it is using AppData\Roaming and not AppData\Local as in many corporates with roaming profile this also gets sync'ed across machines? AppData\Local is also usually open as well (it is used by Click Once deployments).

**Comment by zanderz** [comment]
Not really, only that I worked at BitTorrent previously, and figured that if %APPDATA% is good enough for a billion uTorrent/BitTorrent installations, it would be good enough for us. But times change, and if everyone is locking down %appdata% maybe we should switch.

**Comment by nicstrong** [comment]
%APPDATA_LOCAL% seem more appropriate in this case, as it avoids it being synced with roaming profiles which would cause issues I am guessing (different versions across machines).
Anyway, another colleague had a similar issue with Spotify installer, he got around it using NTFS links i'll try something similar.
Close the issue if you want.
